rtos/mbed_lib.json

```diff
@@ -1,6 +1,198 @@
 {
     "name": "rtos",
     "config": {
-        "present": 1
+        "present": 1,
+
+        "taskcnt": {
+            "help": [
+                "Number of concurrent running user threads <1-250>",
+                " Defines max. number of user threads that will run at the same time.",
+                " Default: 6"
```

Comment:
You can enfore the default by specifying `"value": 6` (assuming of course that the default isn't defined somewhere else (and that there's a good reason to define it somewhere else if it is))